app/views/mailings/show.html.slim

```diff
@@ -1,34 +1 @@
-nav.page
-  .back-nav
-    ul
-      li = icon('chevron-left')
-      li = link_to 'All mailings', mailings_path, class: 'back'
-
-- if can?(:crud, Submission)
-  nav.actions
-    ul.list-inline
-      li = link_to 'Edit', edit_mailing_path(@mailing), class: 'btn btn-default btn-sm edit'
-      li = link_to 'Delete', mailing_path(@mailing), method: :delete, class: 'btn btn-default btn-sm destroy', data: { confirm: 'This action cannot be undone. Are you sure?' }
-
-h1 class="mailing" = @mailing.subject
-
-dl.dl-horizontal
-  dt Seasons
-  dd = @mailing.seasons.any? ? @mailing.seasons.join(', ') : '-'
-  dt Group
-  dd = @mailing.group.humanize
-  dt To
-  dd = Array(@mailing.to).map(&:capitalize).join(', ')
-  dt Recipients count
-  dd = @mailing.recipients.users.count
-  dt From
-  dd = @mailing.from
-  dt Sent at
-  dd = @mailing.sent_at ? @mailing.sent_at.to_formatted_s(:short) : '-'
-
-.mailing
-  .body
-    = render_markdown(@mailing.body).html_safe
-
-- if can?(:read, Submission)
-  = render partial: 'submissions', locals: { mailing: @mailing, submissions: @mailing.submissions }
+= render 'mailings/details', namespace: nil, mailings: @mailings
```

Comment:
Same as for the `index` view: if this partial ain't going to be reused for another mailer view, I'd suggest to simply put the view code in the `show` view.